addons/l10n_ee/data/account_tax_group_data.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data noupdate="1">
-        <record id="tax_group_vat_20" model="account.tax.group">
-            <field name="name">VAT 20%</field>
+        <record id="tax_group_vat_22" model="account.tax.group">
```

Comment:
By renaming the xml_id, you will create a brand new tax group instead of changing an existing one. Is it intended? It means the existing "20%" tax group will remaing with the current tax closing entry configuration and you will not inherit anything from it.

About the update, the file is an `noupdate` block. So, it won't be updated.

Reply:
I just reordered the upper two groups. Nothing changed, except for the new groups. (I can revert that if needed)
I guess it's not a big deal that existing DB's won't be updated, but then new DB's can have them at least. It's just a convenience if they need to use the special taxes, since I already got the question twice.